integration/action-test.ts

```diff
@@ -254,56 +262,54 @@ describe("actions", () => {
   // TODO: figure out what the heck is wrong with this test...
   // For some reason the error message is "Unexpect Server Error" in the test
   // but if you try the app in the browser it works as expected.
-  it.skip("rejects too big of an upload with JavaScript", async () => {
+  test.skip("rejects too big of an upload with JavaScript", async ({
```

Comment:
Did you try this test to see whether using playwright magically fixed it?

Reply:
i did and it didn't 😢